base/applications/osk/settings.h

```diff
@@ -0,0 +1,18 @@
+﻿/*
+ * PROJECT:         ReactOS On-Screen Keyboard
+ * LICENSE:         GPL - See COPYING in the top level directory
```

Comment:
Please use the new license header: https://reactos.org/wiki/Coding_Style#File_Structure